Community/create_a_record/README.txt

```diff
@@ -0,0 +1,37 @@
+Copyright 2019 BlueCat Networks (USA) Inc. and its affiliates
```

Comment:
so it seems like you already have a pull request for create_a_record in https://github.com/bluecatlabs/gateway-workflows/pull/19 , not sure why this is in this pull request